assignments/arrays.js

```diff
@@ -63,32 +66,75 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+var item = inventory.find(item => item.id === 33);
```

Comment:
Consider using 'let' here over 'var'. 'Var' is absolutely valid, but consider reprocussions within an existing code base. Would this have the potential to allow you to declare 'item' for a completely different block of code? How would that affect the existing code?